dashboard/common/db_util.py

```diff
@@ -21,16 +23,16 @@ def canvas_id_to_incremented_id(canvas_id):
     return int(canvas_id) + settings.CANVAS_DATA_ID_INCREMENT
 
 
-def incremented_id_to_canvas_id(incremented_id):
+def incremented_id_to_canvas_id(incremented_id: Union[int, str]) -> Optional[int]:
     try:
         int(incremented_id)
     except ValueError:
         return None
 
-    return str(int(incremented_id) - settings.CANVAS_DATA_ID_INCREMENT)
+    return int(incremented_id) - settings.CANVAS_DATA_ID_INCREMENT
 
 
-def get_course_name_from_id(course_id):
+def get_course_name_from_id(course_id: Union[int, str]) -> str:
```

Comment:
How would you feel about just deleting this function? It's never used in `dashboard`, and it seems like something the ORM should be able to do with a simple filter.

Reply:
Looks like Justin removed the use of this in #535. I can just remove.